drawBot/context/gifContext.py

```diff
@@ -43,7 +44,13 @@ def _writeDataToFile(self, data, path, options):
             options["multipage"] = True
             tempPath = tempfile.mkstemp(suffix=".gif")[1]
 
-        inputPaths = super(GIFContext, self)._writeDataToFile(data, tempPath, options)
+        self._inputPaths = []
+        super(GIFContext, self)._writeDataToFile(data, tempPath, options)
```

Comment:
```suggestion
        super()._writeDataToFile(data, tempPath, options)
```

Reply:
the old style `super(class, self)` happens in more places, will change this one, but the rest has to go in a separate PR

#453